src/ManualSetupPushController.js

```diff
@@ -178,31 +178,35 @@ export default FormController.extend({
         }),
         FormType.View({
           View: View.extend({
-            template: hbs(
-              '\
-                <p class="okta-form-subtitle o-form-explain text-align-c">\
-                  {{instructions}}\
-                </p>\
-              '
-            ),
+            className: 'secret-key-instructions',
+            attributes: { 'data-se': 'secret-key-instructions'},
+            template: hbs`
+            <section aria-live="assertive">
+              <p class="sr-only">{{i18n code="enroll.totp.sharedSecretInstructions.aria.intro" bundle="login"}}</p>
+              <ol>
+                <li>{{step1Instruction}}</li>
+                <li>{{i18n code="enroll.totp.sharedSecretInstructions.step2" bundle="login"}}</li>
+                <li>{{i18n code="enroll.totp.sharedSecretInstructions.step3" bundle="login" 
+                $1="<strong>$1</strong>" $2="<strong>$2</strong>"}}</li>
+                <li>{{i18n code="enroll.totp.sharedSecretInstructions.step4" bundle="login"}}</li>
+              </ol>
+              <p class="shared-key margin-top-10" tabindex=0 
+                aria-label="{{i18n code="enroll.totp.sharedSecretInstructions.aria.secretKey" bundle="login"}} 
+                {{sharedSecretKey}}">{{sharedSecretKey}}</p>
```

Comment:
`{{sharedSecretKey}}` is repeated, I think you only need it once right? it shouldn't be part of the `aria-label` value

Reply:
one is for aria label and another one to display in UI

Reply:
if that is the case, then it should be an argument for `enroll.totp.sharedSecretInstructions.aria.secretKey`. That also helps to have proper translations